recipes/requests_cache/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "requests_cache" %}
+{% set version = "0.4.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name | replace("_", "-" ) }}-{{ version }}.tar.gz
+  sha256: fe561ca119879bbcfb51f03a35e35b425e18f338248e59fd5cf2166c77f457a2
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - requests >=1.1.0
+
+test:
+  imports:
+    - requests_cache
+    - requests_cache.backends
+    - requests_cache.backends.storage
+
+about:
+  home: https://github.com/reclosedev/requests-cache
+  license: BSD
```

This should specify the version of the license:

```suggestion
  license: BSD 2-clause
```